lib/spells/CSpellHandler.cpp

```diff
@@ -1017,55 +1017,4 @@ std::vector<bool> CSpellHandler::getDefaultAllowed() const
 	}
 
 	return allowedSpells;
-}
-
-void CSpellHandler::update780()
```

Comment:
It'll conflict with new-terrains branch. Can you rebase you changes over new-terrains?

Reply:
Alright, will get to it later.